CoroutineTemplate/build.gradle

```diff
@@ -18,11 +18,15 @@ buildscript {
 
         constraint_layout_version = '2.0.0-alpha3'
 
-        hilt_version = '2.33-beta'
+        hilt_version = '2.31.2-alpha'
+        hilt_androidx_version = '1.0.0-alpha01'
```

Comment:
Should we consider upgrading to a (stable) version?

```
hilt_version = '2.35'
```

If so, then we can remove the following dependencies 🚀 :

- `androidx.hilt:hilt-lifecycle-viewmodel`
- `androidx.hilt:hilt-compiler`


Then there's one additional change we need to make in `HomeViewModel`:

```
@HiltViewModel
class HomeViewModel @Inject constructor(
...
```

Reference: https://dagger.dev/hilt/view-model.html

Reply:
I will check more and update 🙏